website/docs/d/iam_trusted_profile_identities.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile_identities"
+description: |-
+  Get information about iam_trusted_profile_identities
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile_identities
+
+Provides a read-only data source for iam_trusted_profile_identities. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_iam_trusted_profile_identities" "iam_trusted_profile_identities" {
+	profile_id = "profile_id"
```

Comment:
Can we provide some valid dummy values or can we use some interpolation instead to fetch the "profile_id" from other resource/datasource
Holds for other doc files also

Reply:
changes have been accomodated.